Storage/tests/System/StreamWrapper/UrlStatTest.php

```diff
@@ -89,7 +89,7 @@ public function testIsExecutable()
     {
         // php returns false for is_executable if the file is a directory
         // https://github.com/php/php-src/blob/master/ext/standard/filestat.c#L907
-        $this->assertFalse(is_executable(self::$dirUrl));
+        $this->assertTrue(is_executable(self::$dirUrl));
```

Comment:
The comment above says 

```
 php returns false for is_executable if the file is a directory
```

As `is_executable` is being run on a directory url (e.g. `$dirUrl`), this should return false. It seems like there's a deeper issue here.

Reply:
In my system and in Kokoro, `is_executable` gives `true` for a given directory
<img width="559" alt="Screenshot 2023-01-04 at 21 13 50" src="https://user-images.githubusercontent.com/7369612/210593323-3a63407a-568a-4701-bf2b-87dcf13966cd.png">

Reply:
Hmm, the comments for [is_executable](https://www.php.net/manual/en/function.is-executable.php) suggest it always returns false on windows. Maybe that's what the comment is implying? 

Looking at the link in the comment (https://github.com/php/php-src/blob/master/ext/standard/filestat.c#L907), I don't see any indication that it always returns false. So maybe switching this is the right thing to do, although we should also remove the comment.